scintilla/lexilla/include/SciLexer.h

```diff
@@ -2058,6 +2062,93 @@
 #define SCE_GD_WORD2 14
 #define SCE_GD_ANNOTATION 15
 #define SCE_GD_NODEPATH 16
+#define SCE_TOML_DEFAULT 0
+#define SCE_TOML_COMMENT 1
+#define SCE_TOML_IDENTIFIER 2
+#define SCE_TOML_KEYWORD 3
+#define SCE_TOML_NUMBER 4
+#define SCE_TOML_TABLE 5
+#define SCE_TOML_KEY 6
+#define SCE_TOML_ERROR 7
+#define SCE_TOML_OPERATOR 8
+#define SCE_TOML_STRING_SQ 9
+#define SCE_TOML_STRING_DQ 10
+#define SCE_TOML_TRIPLE_STRING_SQ 11
+#define SCE_TOML_TRIPLE_STRING_DQ 12
+#define SCE_TOML_ESCAPECHAR 13
+#define SCE_TOML_DATETIME 14
+#define SCE_TROFF_DEFAULT 0
```

Comment:
troff ??

Reply:
Yes, this is `SciLexer.h` taken from lexilla - it contains all the styles for all the languages that lexilla supports (we don't patch it in any way). In addition to Zig, Dart, and TOML that I added to lexilla, the troff lexer was merged independently of me so there's this extra language as well.

Reply:
Ok